feature/gribi/otg_tests/ack_in_presence_other_routes/route_ack_test.go

```diff
@@ -227,21 +227,16 @@ func configureNetworkInstance(t *testing.T) {
 }
 
 // configStaticRoute configures a static route.
-func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) *telemetry.NetworkInstance_Protocol_Static {
-	d := &telemetry.Device{}
-	ni1 := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
-	ni1.Enabled = ygot.Bool(true)
-	ni1.Name = ygot.String(*deviations.DefaultNetworkInstance)
-	ni1.Description = ygot.String("Static route added by gNMI-OC")
-	static := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC")
+func configStaticRoute(t *testing.T, dut *ondatra.DUTDevice, prefix string, nexthop string) *telemetry.NetworkInstance_Protocol {
+	ni1 := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Get(t)
+	static := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "DEFAULT")
 	static.Enabled = ygot.Bool(true)
 	static.Identifier = telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC
```

Comment:
Since this is set in GetOrCreateProtocol, is this necessary?

Reply:
This is inherited from the ATE test. See the original pull #198

Reply:
Let's update both in this patch to keep the files consistent, especially since the changes here are small.

Reply:
Done